# HG changeset patch
# Parent f73b7e96c0735448035ec4786f37dcf7d7644e00
# User Timothy B. Terriberry <tterribe@vt.edu>
Patch to fix errors including C headers in C++

diff --git a/media/libvpx/vp8/common/onyx.h b/media/libvpx/vp8/common/onyx.h
--- a/media/libvpx/vp8/common/onyx.h
+++ b/media/libvpx/vp8/common/onyx.h
@@ -55,24 +55,24 @@ extern "C"
 
     typedef enum
     {
         MODE_REALTIME       = 0x0,
         MODE_GOODQUALITY    = 0x1,
         MODE_BESTQUALITY    = 0x2,
         MODE_FIRSTPASS      = 0x3,
         MODE_SECONDPASS     = 0x4,
-        MODE_SECONDPASS_BEST = 0x5,
+        MODE_SECONDPASS_BEST = 0x5
     } MODE;
 
     typedef enum
     {
         FRAMEFLAGS_KEY    = 1,
         FRAMEFLAGS_GOLDEN = 2,
-        FRAMEFLAGS_ALTREF = 4,
+        FRAMEFLAGS_ALTREF = 4
     } FRAMETYPE_FLAGS;
 
 
 #include <assert.h>
     static void Scale2Ratio(int mode, int *hr, int *hs)
     {
         switch (mode)
         {
diff --git a/media/libvpx/vp8/encoder/onyx_int.h b/media/libvpx/vp8/encoder/onyx_int.h
--- a/media/libvpx/vp8/encoder/onyx_int.h
+++ b/media/libvpx/vp8/encoder/onyx_int.h
@@ -158,17 +158,17 @@ typedef enum
     THR_NEW1           = 13,
     THR_NEW2           = 14,
     THR_NEW3           = 15,
 
     THR_SPLIT1         = 16,
     THR_SPLIT2         = 17,
     THR_SPLIT3         = 18,
 
-    THR_B_PRED         = 19,
+    THR_B_PRED         = 19
 }
 THR_MODES;
 
 typedef enum
 {
     DIAMOND = 0,
     NSTEP = 1,
     HEX = 2
diff --git a/media/libvpx/vpx/vp8.h b/media/libvpx/vpx/vp8.h
--- a/media/libvpx/vpx/vp8.h
+++ b/media/libvpx/vpx/vp8.h
@@ -41,34 +41,34 @@ enum vp8_com_control_id
     VP8_SET_REFERENCE           = 1,    /**< pass in an external frame into decoder to be used as reference frame */
     VP8_COPY_REFERENCE          = 2,    /**< get a copy of reference frame from the decoder */
     VP8_SET_POSTPROC            = 3,    /**< set the decoder's post processing settings  */
     VP8_SET_DBG_COLOR_REF_FRAME = 4,    /**< set the reference frames to color for each macroblock */
     VP8_SET_DBG_COLOR_MB_MODES  = 5,    /**< set which macro block modes to color */
     VP8_SET_DBG_COLOR_B_MODES   = 6,    /**< set which blocks modes to color */
     VP8_SET_DBG_DISPLAY_MV      = 7,    /**< set which motion vector modes to draw */
     VP8_COMMON_CTRL_ID_MAX,
-    VP8_DECODER_CTRL_ID_START   = 256,
+    VP8_DECODER_CTRL_ID_START   = 256
 };
 
 /*!\brief post process flags
  *
  * The set of macros define VP8 decoder post processing flags
  */
 enum vp8_postproc_level
 {
     VP8_NOFILTERING             = 0,
     VP8_DEBLOCK                 = 1<<0,
     VP8_DEMACROBLOCK            = 1<<1,
     VP8_ADDNOISE                = 1<<2,
     VP8_DEBUG_TXT_FRAME_INFO    = 1<<3, /**< print frame information */
     VP8_DEBUG_TXT_MBLK_MODES    = 1<<4, /**< print macro block modes over each macro block */
     VP8_DEBUG_TXT_DC_DIFF       = 1<<5, /**< print dc diff for each macro block */
     VP8_DEBUG_TXT_RATE_INFO     = 1<<6, /**< print video rate info (encoder only) */
-    VP8_MFQE                    = 1<<10,
+    VP8_MFQE                    = 1<<10
 };
 
 /*!\brief post process flags
  *
  * This define a structure that describe the post processing settings. For
  * the best objective measure (using the PSNR metric) set post_proc_flag
  * to VP8_DEBLOCK and deblocking_level to 1.
  */
diff --git a/media/libvpx/vpx/vp8cx.h b/media/libvpx/vpx/vp8cx.h
--- a/media/libvpx/vpx/vp8cx.h
+++ b/media/libvpx/vpx/vp8cx.h
@@ -172,17 +172,17 @@ enum vp8e_enc_control_id
      * per-frame bitrate, with the special (and default) value 0 meaning
      * unlimited, or no additional clamping beyond the codec's built-in
      * algorithm.
      *
      * For example, to allocate no more than 4.5 frames worth of bitrate
      * to a keyframe, set this to 450.
      *
      */
-    VP8E_SET_MAX_INTRA_BITRATE_PCT,
+    VP8E_SET_MAX_INTRA_BITRATE_PCT
 };
 
 /*!\brief vpx 1-D scaling mode
  *
  * This set of constants define 1-D vpx scaling modes
  */
 typedef enum vpx_scaling_mode_1d
 {
@@ -253,17 +253,17 @@ typedef enum
  *
  */
 
 typedef enum
 {
     VP8_ONE_TOKENPARTITION   = 0,
     VP8_TWO_TOKENPARTITION   = 1,
     VP8_FOUR_TOKENPARTITION  = 2,
-    VP8_EIGHT_TOKENPARTITION = 3,
+    VP8_EIGHT_TOKENPARTITION = 3
 } vp8e_token_partitions;
 
 
 /*!\brief VP8 model tuning parameters
  *
  * Changes the encoder to tune for certain types of input material.
  *
  */
